exporter/elasticsearchexporter/README.md

```diff
@@ -39,7 +39,7 @@ As a shortcut, the following settings are also supported:
 
 - `user` (optional): Username used for HTTP Basic Authentication.
 - `password` (optional): Password used for HTTP Basic Authentication.
-- `api_key` (optional): [Elasticsearch API Key] in "encoded" format.
+- `api_key` (optional): [Elasticsearch API Key] in "encoded" format (e.g. `VFR2WU41VUJIbG9SbGJUdVFrMFk6NVVhVDE3SDlSQS0wM1Rxb24xdXFldw==`).
```

Comment:
safe to assume this key is not valid?

Reply:
definitely not valid, thanks for checking!